scripts/analysis_tests.py

```diff
@@ -0,0 +1,10 @@
+import unittest
+
+class test_analysis(unittest.TestCase):
+
+	def test_get_cursor(self):
+
```

Comment:
why so many blank lines?

Reply:
sorry this was a duplicate file I created forgetting that I already had one for analysis.